Gopkg.lock

```diff
@@ -1334,6 +1334,7 @@
     "k8s.io/helm/pkg/renderutil",
     "k8s.io/helm/pkg/timeconv",
     "k8s.io/klog",
+    "sigs.k8s.io/yaml",
```

Comment:
Now that we have a direct dependency on `sigs.k8s.io/yaml`, should we add it to `Gopkg.yaml` so that we can lock the version?

```
[[constraint]]
  name = "github.com/kubernetes-sigs/yaml"
  version = "v1.1.0"
```

Reply:
Unless we strictly need to pin to a version, I'd rather let `sigs.k8s.io/yaml` float with the version `client-go`/`kubernetes` pulls in. We previously weren't specifying `github.com/ghodss/yaml` in `Gopkg.toml`. Hopefully this gets merged back into `github.com/ghodss/yaml` at some point.

Reply:
Oh that makes sense to let `client-go`/`kubernetes` pull it in. Sounds good!